examples/fluids/qfunctions/newtonian.h

```diff
@@ -783,4 +784,28 @@ CEED_QFUNCTION(PressureOutflow_Jacobian_Prim)(void *ctx, CeedInt Q,
   return PressureOutflow_Jacobian(ctx, Q, in, out, StateFromY, StateFromY_fwd);
 }
 
+// *****************************************************************************
+// This QFunction implements the Harten Lax VanLeer (HLL) approximate Riemann
+// solver. Taking in two states.
+// *****************************************************************************
+CEED_QFUNCTION_HELPER int Harten_Lax_VanLeer_Flux(void *ctx, CeedInt Q,
+    const CeedScalar *const *in, CeedScalar *const *out,
+    StateFromQi_t StateFromQi, StateFromQi_fwd_t StateFromQi_fwd) {
+  // *INDENT-OFF*
+  // Inputs will be uleft and uright states
+
+  // Output will be the flux
+
+  // *INDENT-ON*
+  // Context
+
+  CeedPragmaSIMD
+  for (CeedInt i=0; i<Q; i++) {
+
+  } // End of HLL flux
```

Comment:
```suggestion
  StateConservative flux_left[3], flux_right[3]; // fluxes in 3 dimensions
  FluxInviscid(gas, left, flux_left);
  FluxInviscid(gas, right, flux_right);
  // Now compute flux * normal.
```
You can make another small helper to get flux dot normal.